tingbot/platform_specific/sdl_wrapper.py

```diff
@@ -0,0 +1,66 @@
+import pygame
+import os
+from ..graphics import Surface,color_map
+
+button_callback = None
+
+class Wrapper(Surface):
+    def __init__(self):
+        self.surface = pygame.display.set_mode((470, 353))
+        background = pygame.image.load(os.path.join(os.path.dirname(__file__), 'bot.png'))
+        self.surface.blit(background,(0,0))
+        self.screen = self.surface.subsurface((60,40,320,240))
+        xPositions = (60, 100, 320, 360)
+        self.buttons = []
+        for x in range(4):
+            self.buttons.append(Button(self.surface.subsurface(xPositions[x],0,22,12),x))
+
+    def get_screen(self):
```

Comment:
I think this is a hangover from when I had things arranged differently. I can just access the screen directly, so I agree it is not needed.

Will fix when I get home tonight